gdcdictionary/schemas/diagnosis.yaml

```diff
@@ -525,44 +495,30 @@ properties:
       - Not Reported
       - Not Allowed To Collect
 
-  hiv_status:
+  hiv_positive:
     term:
-      $ref: "_terms.yaml#/hiv_status"
+      $ref: "_terms.yaml#/hiv_positive"
     enum:
-      - Equivocal
-      - Indeterminate
-      - Not Done
-      - Not Available
-      - Negative
-      - Positive
+      - "Yes"
```

Comment:
We've had this type of yes/no/unknown structure in many previous fields. How would it look as a boolean? Would we be able to capture "Unknown"?